### PR TITLE
backport 5.0: De-register first from SCC or RMT/SMT (#3415)

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/23002
https://bugzilla.suse.com/show_bug.cgi?id=1216946

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+- Added reminder note to unregister before registration to Client
```
Comment (keichwa) [inline CHANGELOG.md:1]:
```suggestion
- Added reminder note to de-register before registration to Client
```

**Comment by 0rnela** [comment]
@keichwa -- same suggestions would need to go to the master branch too

File: modules/client-configuration/pages/registration-overview-suse.adoc [hunk 2]
```diff
@@ -19,9 +19,27 @@ You must also have created an activation key.
...
+
+
+
+[[suse-registration-overview-check]]
+== Check Previous Registration Status
+
+If the client was registered to SCC, SMT, or RMT previously, remove such a registration with [command]``SUSEConnect`` before migrating the client to {productname} Server.
+If you forget about removing the old registration first, it can happen that repositories not needed anymore getting automatically added to the client even after the client has been migrated to a newer version of a {suse} operating system using {productname}.
+
+To de-register a {sle} 12 or 15 client, on the client, as root, execute:
```
[the remark this answers is not in the document]
Comment (keichwa) [reply modules/client-configuration/pages/registration-overview-suse.adoc:38]: I'm not that sure. We often use ``..., as root, ...`` (Lana coined that phrase).  You can try:
```
uyuni-docs/modules> grep -e ', as.*root,' */pages/*adoc
```

File: modules/client-configuration/pages/registration-overview-suse.adoc [hunk 3]
```diff
@@ -19,9 +19,27 @@ You must also have created an activation key.
...
 For more information about using multiple servers, see xref:specialized-guides:large-deployments/hub-multi-server.adoc[].
 ====
+
+
+
+[[suse-registration-overview-check]]
+== Check Previous Registration Status
+
+If the client was registered to SCC, SMT, or RMT previously, remove such a registration with [command]``SUSEConnect`` before migrating the client to {productname} Server.
+If you forget about removing the old registration first, it can happen that repositories not needed anymore getting automatically added to the client even after the client has been migrated to a newer version of a {suse} operating system using {productname}.
```
[the remark this answers is not in the document]
Comment (keichwa) [reply modules/client-configuration/pages/registration-overview-suse.adoc:36]: thanks, I'd like to improve master first.  Please, review https://github.com/uyuni-project/uyuni-docs/pull/3422 .

**Pull request merged**